src/CMakeLists.txt

```diff
@@ -3,13 +3,23 @@ project(ki017)
 
 set(SOURCE
         main.cpp
+		AI.cpp
+		AICallback.cpp
+		util/Logging.cpp
+		generate/ItemChoice.cpp
+		generate/EquipmentChoice.cpp
+		generate/GameOperation.cpp
+		generate/random/ItemChoice.cpp
+		generate/random/EquipmentChoice.cpp
+		generate/random/GameOperation.cpp
         )
 
 add_executable(${PROJECT_NAME} ${SOURCE})
 target_compile_options(${PROJECT_NAME} PRIVATE ${COMMON_CXX_FLAGS} -Wall -Wextra -Wpedantic -Werror -mtune=native -march=native)
 target_link_libraries(${PROJECT_NAME}
 		spdlog::spdlog
 		CLI11::CLI11 stdc++fs
+		SopraNetwork
```

Comment:
Ist das netzwerk nicht komplett im LibClient enthalten?

Reply:
Eigentlich schon, aber aus irgendwelchen Gründen findet der die Header sonst nicht.

Reply:
fix: SopraNetwork muss in LibClient pulic und nicht private gelinkt werden (-> #26 )